scrape.py

```diff
@@ -339,7 +339,7 @@ def add_linux_ami_info(instances):
     # Some background info at https://github.com/powdahound/ec2instances.info/pull/161
     for i in instances:
         i_family_id = i.instance_type.split('.')[0]
-        if i_family_id in ('cc2', 'cg1', 'hi1', 'hs1'):
+        if i_family_id in ('cc2', 'cg1', 'hi1', 'hs1', 'i3en'):
```

Comment:
Hmm, now this is interesting. It seems there's virtualization missing for many more instances that just i3en. I think that instead of adding something to this old hack we should rather look into why the info is not set for some instances. Also we can probably set HVM as default since PV instances are very old.